core/src/main/java/org/apache/carbondata/core/indexstore/row/DataMapRow.java

```diff
@@ -26,70 +26,66 @@
  */
 public abstract class DataMapRow implements Serializable {
 
-  protected CarbonRowSchema[] schemas;
```

Comment:
I don't think it is required to remove the schema from here as it is a temporary object to read data. It does not occupy any heap memory.  Many changes related to this is not required. Please revert the changes